object_storage_buckets.go

```diff
@@ -15,12 +15,17 @@ import (
 type ObjectStorageBucket struct {
 	Label string `json:"label"`
 
-	// Only one of Region or Cluster will be provided with a non-empty value,
-	// depending on your account.
+	// Deprecated: Cluster field has been deprecated.
+	//
+	// Please consider switching to use the 'Region' field.
+	// If your Cluster is `us-mia-1`, then the region would be `us-mia`.
+	//
+	// The 'Cluster' field doesn't necessarily reflect the accurate cluster
+	// the bucket is in if there are multiple clusters in the region.
+	// It will always be `<REGION>-1` (e.g., `us-mia-1`) for backward
+	// compatibility purposes.
```

Comment:
Minor wording changes 🙂 
```
Deprecated: The Cluster field has been deprecated.

Going forward, the Region field will be the supported way to designate where Object Storage Buckets live. This is necessary because there are now multiple Object Storage clusters to a region. For example, a Cluster value of `us-mia-1` will translate to a Region value of `us-mia`.

NOTE: The 'Cluster' field will always return a value similar to `<REGION>-1` (e.g., `us-mia-1`) for backward compatibility purposes.
```